modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -681,7 +682,9 @@ private void updateAssignmentInternal(ConfigurationNotificationEvent<byte[]> ass
 
                 String grpId = partitionRaftGroupName(tblId, partId);
 
-                placementDriver.updateAssignment(grpId, nodes);
+                TablePartitionId replicaGrpId = new TablePartitionId(tblId, partId);
```

Comment:
`String grpId = partitionRaftGroupName(tblId, partId);` should be removed.

Reply:
I removed this method, but I think, it isn't exactly what you expected, because now I am using ReplicationGroupId#toString() instead of them.

Reply:
Yep, answered above.